crates/lib/src/store/mod.rs

```diff
@@ -35,8 +35,10 @@ use crate::spec::ImageStatus;
 use crate::utils::deployment_fd;
 
 /// See https://github.com/containers/composefs-rs/issues/159
+// pub type ComposefsRepository =
+//    composefs::repository::Repository<composefs::fsverity::Sha512HashValue>;
 pub type ComposefsRepository =
```

Comment:
Per https://github.com/bootc-dev/bootc/issues/1190 I think we should change all of the code to use this alias instead